python/pyarrow/tests/test_array.py

```diff
@@ -714,7 +714,8 @@ def test_struct_from_arrays():
         pa.StructArray.from_arrays(
             arrays, fields, mask=pa.chunked_array([mask]))
 
-    # Non-empty array with no fields https://github.com/apache/arrow/issues/15109
```

Comment:
Nit: Looks like an unnecessary change

Reply:
pylint was complaining for me.  But, oddly, it was complaining about an 80 character line limit and I thought we allowed 90.  I'll revert and see if CI passes.  Maybe I have a stray .pylintrc in my environment somewhere.

Reply:
Ah, my install of archery was out of date it seems.  I updated it and the warning went away.  I suspect we changed where we store the flake config.

Reply:
Thanks for the catch

Reply:
I believe PyArrow allows 87 chars at most. Thanks for fixing 👍🏻